Makefile.am

```diff
@@ -218,6 +218,9 @@ endif
 check-formatting:
 	$(MAKE) -C src $@
 
+electrs: ${PWD}/src/electrs
```

Comment:
PWD is not necessarily the out-of-source dir, use top_builddir.  And if ${PWD}/src/electrs does not exist, there's no rule to make it.  Instead use a phony target that points to the real target like this:

.PHONY: electrs

electrs: $(top_builddir)/src/electrs

$(top_builddir)/src/electrs:
	$(top_srcdir)/contrib/electrs/build_electrs.py --dst=$(top_builddir)/src

Reply:
fixed in 81000427bb55991ecdc35bf4509f12cb287af3ac